2.2/root/opt/app-root/etc/puma.cfg

```diff
@@ -9,6 +9,11 @@ def get_max_memory()
     (2 ** (8*0.size - 2) - 1)
 end
 
+def per_worker_memory()
+    # Unless told otherwise, allow 256MiB per worker
+    ENV.fetch('MEMORY_BYTES_PER_WORKER', '268435456').to_i
```

Comment:
Seems this shloud be a constant specified somewhere else and not in string. 
Writing it as `256*(2**20)` makes more sense IHMO.

But I am suspicios about this constant altogether. It suits your use, but doubt it's universal.